bokehjs/test/models/tools/edit/point_draw_tool.coffee

```diff
@@ -0,0 +1,205 @@
+{expect} = require "chai"
```

Comment:
Are there any reasons not to write this in TypeScript?

Reply:
No, happy to rewrite it, I thought you might be sticking with CoffeeScript in tests for now since I could not find any tests written in TypeScript. Guess it's just one of the last things you'll convert.

Reply:
There aren't many, but should be sufficient for inspiration:
```
$ git ls-files bokehjs/test/ | grep 'ts$'
test/client/index.ts
test/core/has_props.ts
test/core/index.ts
test/core/selector.ts
test/core/signaling.ts
test/core/util/array.ts
test/core/util/refs.ts
test/defaults.ts
test/models/tickers/adaptive_ticker.ts
test/models/tickers/categorical_ticker.ts
test/models/tickers/composite_ticker.ts
test/models/tickers/continuous_ticker.ts
test/models/tickers/fixed_ticker.ts
test/models/tickers/index.ts
test/models/tickers/log_ticker.ts
test/models/tickers/mercator_ticker.ts
test/models/tickers/single_interval_ticker.ts
test/protocol/index.ts
test/size.ts
```
Unfortunately I can't yet rewrite the rest of the tests in bulk, only those which have solid typings.

Reply:
Great thanks, I did an initial conversion but will use those to make the style a bit more consistent.

Reply:
btw. Tests are dynamically compiled, so you have to actually run tests to see any compilation errors.